### PR TITLE
Advancing 0.19.0-rc3 release to status: admiral

### PR DESCRIPTION
**Pull request by Jaanki**
Components:
* https://github.com/submariner-io/admiral/commits/release-0.19
* https://github.com/submariner-io/shipyard/commits/release-0.19
Depends on https://github.com/submariner-io/admiral/pull/1009

### CODE REVIEW

**Review by Jaanki — COMMENTED** [review]
This is wrong SHA. The correct one should be 8a5ecb962986f6d57d43845e8a3bea8b172f508f as per https://github.com/submariner-io/admiral/commits/release-0.19/

**Review by vthapar — APPROVED** [review]

**Comment by vthapar** [comment]
Missed your comment. Are you pushing followup commit with correct sha?

**Review by tpantelis — APPROVED** [review]

File: releases/v0.19.0-rc3.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 version: v0.19.0-rc3
 name: 0.19.0-rc3
 branch: release-0.19
-status: shipyard
+status: admiral
 components:
   shipyard: 35f3468294128ce32bec0d775ff3131127dd9076
+  admiral: 99b5ac397a07a2256f4fa88ef99a5e65f744e149
```
Comment (vthapar) [inline releases/v0.19.0-rc3.yaml:8]:
```suggestion
  admiral: 8a5ecb962986f6d57d43845e8a3bea8b172f508f
```

**Review by vthapar — CHANGES_REQUESTED** [review]

**Comment by Jaanki** [comment]
Pushing a new PR with correct SHA.

**Pull request closed without merging**